.github/workflows/ci-merge-queue.yml

```diff
@@ -51,6 +52,7 @@ jobs:
     name: Acquire Base Image
     needs: save-docker-login-token
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
This timeout for acquire base image is shorter than those in `ci-main` or `ci-pr-forks`, because by the time a PR is merged to main, there's already a base image existing and we don't need to build it?

Reply:
I was thinking CI on main wouldn't ever need to rebuild the build image, but actually, that's not true since it will need to rebuild it for PRs from forks. I'll bring it up to the same value as the others.